context/app/static/js/components/Detail/entityHeader/EntityHeader/utils.js

```diff
@@ -1,29 +1,27 @@
-const extractValuesFromObject = (obj, keys) =>
-  keys.reduce((acc, k) => {
-    if (k in obj) {
-      acc.push(obj[k]);
-    }
+const extractAndLabelMetadata = (obj, keys) => {
+  const labels = { mapped_organ: 'organ type', mapped_data_types: 'data type', mapped_specimen_type: 'specimen type' };
+  return keys.reduce((acc, k) => {
```

Comment:
```
const acc = {};
keys.foreach((k) => {
  acc[k] = { value: k in obj ? obj[k] : undefined, label: k in labels ? labels[k] : k };
});
return acc;
```
I'm not sure that `reduce` is the clearest way to explain what's going on here... but not a strongly held opinion.

Reply:
Done.